CITATION.cff

```diff
@@ -15,7 +15,6 @@ authors:
   affiliation: Center for Astrophysics | Harvard & Smithsonian, CfA, 60 Garden St.,
     02138 Cambridge MA, US
   orcid: https://orcid.org/0000-0003-4568-7005
-  email: "axel.donath@cfa.havard.edu"
```

Comment:
Why does it remove the mail address here?

Reply:
Isn't it support anymore in cff v1.2.0?

Reply:
You're right !
The email attribute is allowed in the cff v1.2.0 schema.
I have recovered it in last commit.